Curation: guidelines for reviewers and approvers (#185)
codewars/docs: content/curating/index.md

---
sidebar: curating
next: /curating/kata/
---

# Content Curation

All content available on Codewars is reviewed, approved, and maintained by members of the Codewars community. If Codewars users want to ensure that working with kata and translations is a pleasant, fun, and educative experience, there's no one else than users themselves who can guarantee that the content is of sufficient quality.

> To support users in assuring good quality of kata and translations, a set of help pages was created with articles related to curating and maintenance activities: tutorials explaining how to review and maintain Codewars content, and reference of guidelines and requirements which a piece of content should meet to be considered good.

Reviewer: ```suggestion
To support users in assuring good quality kata and translations, a set of help pages were created with articles related to curating and maintenance activities: tutorials explaining how to review and maintain Codewars content, and a reference of guidelines and requirements that a piece of content should meet to be considered good.
```

# Content Curation

> All content available on Codewars is reviewed, approved, and maintained by members of Codewars community. If Codewars users want to ensure that working with kata and translations is a pleasant, fun, and educative experience, there's no one else than users themselves who can guarantee that the content is of sufficient quality.

Reviewer: ```suggestion
All content available on Codewars is reviewed, approved, and maintained by members of the Codewars community. If Codewars users want to ensure that working with kata and translations is a pleasant, fun, and educative experience, there's no one else than users themselves who can guarantee that the content is of sufficient quality.
```

To support users in assuring good quality of kata and translations, a set of help pages was created with articles related to curating and maintenance activities: tutorials explaining how to review and maintain Codewars content, and reference of guidelines and requirements which a piece of content should meet to be considered good.

> This section of documentation is meant mostly for reviewers, approvers, and maintainers, who would like to work with existing content created by other users. Kata authors and translators can find helpful documentation in area related to [authoring][docs-authoring].

Reviewer: ```suggestion
This section of the documentation is meant mostly for reviewers, approvers, and maintainers, who would like to work with existing content created by other users. Kata authors and translators can find helpful documentation in areas related to [authoring][docs-authoring].
```
